tools/build_file_generator/third_party_deps/BUILD

```diff
@@ -21,42 +21,41 @@ checkstyle_test(
     target = ":third_party_deps_analyzer",
 )
 
-# '%START%' and '%END%' are markers for 'third_party_deps_updater.sh' script.
-# Don't delete it while 'third_party_deps_updater.sh' script is used.
+# '%START%' and '%END%' are markers for 'third_party_deps_updater.sh' script and ThirdPartyDepsAnalyzer class.
+# Don't delete it while 'third_party_deps_updater.sh' script and ThirdPartyDepsAnalyzer class are used.
 THIRD_PARTY_DEPS_GROUP = [
     # %START%
     "//third_party/maven/com/google/api:api_common",
     "//third_party/maven/com/google/auth:google_auth_library_credentials",
     "//third_party/maven/com/google/auth:google_auth_library_oauth2_http",
-    "//third_party/maven/com/google/auto:auto_common",
     "//third_party/maven/com/google/auto/factory:auto_factory",
     "//third_party/maven/com/google/auto/service:auto_service",
     "//third_party/maven/com/google/auto/service:auto_service_plugin",
     "//third_party/maven/com/google/auto/value:auto_value",
     "//third_party/maven/com/google/auto/value:auto_value_annotations",
     "//third_party/maven/com/google/auto/value:auto_value_plugin",
+    "//third_party/maven/com/google/auto:auto_common",
     "//third_party/maven/com/google/cloud:google_cloud_core",
     "//third_party/maven/com/google/cloud:google_cloud_core_grpc",
     "//third_party/maven/com/google/cloud:google_cloud_firestore",
     "//third_party/maven/com/google/cloud:google_cloud_storage",
-    "//third_party/maven/com/google/code/findbugs:jsr305",
-    "//third_party/maven/com/google/code/gson",
-    "//third_party/maven/com/google/dagger",
+    "//third_party/maven/com/google/code/gson:gson",
+    "//third_party/maven/com/google/dagger:dagger",
     "//third_party/maven/com/google/dagger:dagger_compiler",
     "//third_party/maven/com/google/errorprone:error_prone_ant",
     "//third_party/maven/com/google/firebase:firebase_admin",
-    "//third_party/maven/com/google/flogger",
+    "//third_party/maven/com/google/flogger:flogger",
     "//third_party/maven/com/google/flogger:flogger_system_backend",
     "//third_party/maven/com/google/googlejavaformat:google_java_format",
-    "//third_party/maven/com/google/guava",
-    "//third_party/maven/com/google/jimfs",
+    "//third_party/maven/com/google/guava:guava",
```

Comment:
There's no need to repeat the target if the folder name is the same:
`//third_party/maven/com/google/guava:guava`
Is the same as:
`//third_party/maven/com/google/guava`

Is it easy to shorten them?

Reply:
Added logic to shorten it to the script.